lib/src/modal/modal.component.html

```diff
@@ -0,0 +1,23 @@
+<div
+    class="modal fade"
+    [@openClose]="toggle ? 'open' : 'close'"
+    [ngClass]="modalPosition"
+    tabindex="-1"
+    role="dialog"
+    [attr.id]="id"
+    #modalRef
+>
+    <div class="modal-dialog" [ngClass]="size" [class.modal-dialog-centered]="center" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <ng-content select="[header]"></ng-content>
+            </div>
```

Comment:
would this work instead of wrapping the content??
```
        <ng-content select="[header]" class="modal-header"></ng-content>
```

Reply:
The content might be just text, it's safer to wrap it

Reply:
the problem with this is that the class will not be completely ignored and will not render

Reply:
found a better way with css only, 
```css 
.modal-header:empty {
    display: none;
}
```